src/SayMoreTests/model/ProjectTests.cs

```diff
@@ -347,6 +347,20 @@ public void SetContributorsListToSession_ContributorsInMetaAndSession_AllContrib
 			Assert.AreEqual(4, sessionXml.SelectSingleNode("//participants").InnerText.Split(';').Length, "Expecting four participants (with roles): Iskender Demirel, Lahdo Agirman, Greg Trihus, Eliyo Acar");
 		}
 
+		[Test]
+		[RequiresSTA]
+		public void SetContributorsListAndMigrateAccessProtocolToSession_MigrateAccessProtocol_NewAccessProtocolNodeCreated()
```

Comment:
Looks like this test is for the case where the access protocol does not exist. Another test is needed to test the code path where the access protocol is changed.